test/commands/hset.js

```diff
@@ -24,4 +24,20 @@ describe('hset', () => {
           email: 'bruce@wayne.enterprises',
         })
       ));
+
+  it('should let you set multiple hash map keys and values in a single command', () => {
+    const hash = { id: '2', email: 'bruce@wayne.enterprises' };
+    return redis
+      .hset('user:2', 'id', '2', 'email', 'bruce@wayne.enterprises')
+      .then(status => expect(status).toBe(2))
```

Comment:
This is the failing test 🤔

Reply:
I think the issue is that the `redis` instance is being reused between tests, so `email` is already set for this test.

Reply:
Ah yes, I agree 👍